controllers/modelmesh/model_type_labels_test.go

```diff
@@ -53,14 +57,17 @@ func TestGetServingRuntimeSupportedModelTypeLabelSet(t *testing.T) {
 
 	expectedLabels := []string{
 		"mt:type1",
-		"mt:type3",
+		"mt:type2",
```

Comment:
Should this match l52 as type3?

Reply:
Since `type3` doesn't have the `AutoSelect: true` label, it is not expected to be one of the returned labels. However, `mt:type2` should be included since a type2 model format does have the `AutoSelect: true` label (this wasn't here initially, but probably should have been)

Reply:
Hmm, I am not sure the logic in GetServingRuntimeSupportedModelTypeLabelSet is correct then.

I might misunderstand this, but I thought the following should create one entry as supported model type, including the version number.
{
         Name:       "type2",
         Version:    &version_semver,
         AutoSelect: &autoSelectVal,
},

The current code in model_type_labels.go would create two entries, meaning the version number is irrelevant. Is this expected?